workflows/raredisease.nf

```diff
@@ -406,63 +416,150 @@ workflow RAREDISEASE {
     )
     ch_versions = ch_versions.mix(QC_BAM.out.versions)
 
-    //
-    // EXPANSIONHUNTER AND STRANGER
-    //
-    if (params.analysis_type.equals("wgs")) {
-        CALL_REPEAT_EXPANSIONS (
-            ch_mapped.genome_bam_bai,
-            ch_variant_catalog,
-            ch_case_info,
-            ch_genome_fasta,
-            ch_genome_fai
-        )
-        ch_versions = ch_versions.mix(CALL_REPEAT_EXPANSIONS.out.versions)
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    CALL AND ANNOTATE REPEAT EXPANSIONS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*/
+
+    if (!params.skip_repeat_analysis) {
+        if ( params.analysis_type.equals("wgs") ) {
```

Comment:
Wouldn't it be better to write this in one if statement?
`if (!params.skip_repeat_analysis && params.analysis_type.equals("wgs"))`